src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -723,6 +725,8 @@ private Path discoverSdkPath() {
           // prevent interference from exceptions in other resolvers
           logger.log(Level.SEVERE, resolver.getClass().getName()
               + ": exception thrown when searching for Google Cloud SDK", ex);
+          throw new CloudSdkNotFoundException(
```

Comment:
yeah we probably want to make the comment more clear above the logger to indicate more clearly that we don't want to fail hard here.